app/workers/geocode_application_address_worker.rb

```diff
@@ -0,0 +1,12 @@
+class GeocodeApplicationAddressWorker
+  include Sidekiq::Worker
```

Comment:
Should we define a new queue for geolocation?

Reply:
```
 queue_as :geocoding
```

Reply:
Good idea. Any objection to just using the existing `low_priority` queue? https://github.com/DFE-Digital/apply-for-teacher-training/blob/8b72523670e4e9f707f40d06d183a83ea5a0997d/config/sidekiq.yml#L5

Reply:
Update to use low_priority